idea_town/frontend/static-src/styles/vendor/mozilla-tabzilla/css/tabzilla.scss

```diff
@@ -0,0 +1,34 @@
+#tabzilla {
```

Comment:
Ah, you're right - there is such a gulp task, but the .gitignore is slightly incorrect. I'll fix